### PR TITLE
Add editorial feedback for security article

### PR DESCRIPTION
**Pull request by DonNateR**
- Fixes issue #1535

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b1a9eca41a/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5307da98b7/how-to/)

File: content/cloud-servers/windows-server-security-best-practices.md [hunk 1]
```diff
@@ -69,58 +67,73 @@ exploitation attempts against commonly named accounts or services on the server:
...
+server environment where servers are placed behind physical firewalls and can be
+connected over VPN tunneling through that firewall appliance. 
+
+Rackspace supports a VPN only if it is through a hardware firewall in a solution called
+RackConnect. It is easier to implement this physical firewall setup before you create
+servers because, at the time this article was written, the process that connects the
+firewall and the servers is automated during the build process. Physical firewalls are not
+provisioned as quickly as cloud servers and must be requested through our Hybrid
+teams. For more information about physical firewalls and RackConnect, see
+http://www.rackspace.com/cloud/hybrid/rackconnect/.
```
Comment (nimbinatus) [inline content/cloud-servers/windows-server-security-best-practices.md]: @DonNateR , the link isn't showing as a hyperlink in Nexus.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-028527c406/how-to/)

**Comment by nimbinatus** [comment]
+1

**Pull request merged**